code/game/objects/items/weapons/manuals.dm

```diff
@@ -499,6 +499,165 @@
+
+				<h3>Challengers</h3>
+				<p>The Syndicate trains its novice agents through a Battle Royale where they have to eliminate each others aboard one of our stations. On the bright side, those challengers are here to kill each other. On the downside, they don't care about how much collateral damage they cause, AND the victor may graduate to become a fully fledged Syndicate agent, enabling them to cause even more trouble. We do not want that. Challengers often grimly display the corpse of their target on Newscasters, use this opportunity to try and figure out the identity of the victor so you may deal with them.
+
+				<h3>Elite Strike Team</h3>
+				<p>The Syndicate's strongest soldiers. Those who have often accomplished and survived the destruction of several space stations. Sightings are rare, but they generally seem to field upgraded black rigsuits and extremely lethal ballistic weaponry. Their arrival is almost certain to spell doom for any station in a sector; do what you can to stop or delay them. 
+
+				<h2><img src='data:image/png;base64,[icon2base64(icon('icons/logos.dmi', "vampire-logo"))]' style='position: relative; top: 8;'><a id="Vampire">The Vampire Lords</h2>
+				<p>We didn't want to believe it ourselves at first, but it has come to our attention that some employees have displayed what can only be described as "vampiric powers". We don't know for sure if those awakened here in space, or if they came from Earth and managed to fit among the migrants to our system, but they seem to exhibit a tendency to try and turn our space stations into their own personal kingdoms and blood banks. Some of the powers we've observed include the ability to turn into bats, hypnotize humans into a paralysis, turn those humans into their thralls, and heal all of their wounds even from a dead state.
+                <p>Because of those abilities, conventional ways of fighting prove often inefficient, fortunately, they appear to share the weakness of fairy tale vampires, namely an allergy to holy water, garlic, and mostly, they turn to dust in view of the sun, or in the presence of a deity. A great way to get rid of them permanently is thus to drag them in the station's chapel until their body turns to ashes. Holy Water and Harebells Incense also proves very efficient at harming them and hindering their abilities. Lastly, the Nullrods wielded by chaplains have shown efficiency at protecting them from those same powers.
```

Comment:
```suggestion
                <p>Due to the high cost of silver compared to lead and energy weaponry, Nanotrasen has declined to issue specific anti-vampire bullets. However, vampire lords have shown a surprising weakness to all that is Holy, regardless of creed and denomination. A great way to get rid of them permanently is thus to drag them in the station's chapel until their body turns to ashes. Holy Water and Harebells Incense also proves very efficient at harming them and hindering their abilities. Lastly, the Nullrods wielded by chaplains have shown efficiency at protecting them from those same powers.
```

Reply:
I would also remove all discussion of the specifics of how to kill the vamp. Just say they fear the holy items.

Reply:
I would argue that the specifics of how to kill the vamps are the most important information here.